HealthVault/Classes/Caching/MHVThingCacheDatabaseProtocol.h

```diff
@@ -183,6 +183,28 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)deletePendingMethods:(NSArray<MHVPendingMethod *> *)pendingMethods
                   completion:(void (^)(NSError *_Nullable error))completion;
 
+/**
+ This method is called AFTER a Thing (or Things) have been created and the internet connection is offline. The Thing(s) in the collection should be cached with an identifier that can be used to fetch and delete all 'pending' Things once the internet connection is restored and the database is synced.
```

Comment:
A little unclear,  "This method is called AFTER a Thing (or Things) creation to HealthVault has been attempted, but the internet connection is offline."